src/component/marker/MarkerModel.ts

```diff
@@ -53,6 +53,7 @@ export interface MarkerPositionOption {
     // Absolute position, px or percent string
     x?: number | string
     y?: number | string
+    relativeTo?: 'screen' | 'coordinate'
```

Comment:
`global` sounds good to me, only that it's an adj. instead of noun. In the case of `relativeTo` something, it may be better to follow with a noun. GPT suggests `container` and `coordinateSystem` and I think it's Okay. What do you think?

Reply:
It seems that in programming context global can be used as a noun, which is an abbreviation of "global variable", "global scope", etc. 
But it's OK to use `container`, since no better alternative can be found.